src/openloco/objects/industry_object.h

```diff
@@ -8,7 +8,13 @@ namespace openloco
     struct industry_object
     {
         string_id name;
-        uint8_t pad_02[0xDE - 0x02];
+        uint8_t pad_02[0xCA - 0x02];
+        uint16_t designed; // 0xCA start year
+        uint16_t obsolete; // 0xCC end year
+        uint8_t var_CE;
+        uint8_t cost_ind;  // 0xCF
+        int16_t cost_fact; // 0xD0
+        uint8_t pad_D2[0xDE - 0xD2];
         uint8_t produced_cargo_type[2]; // 0xDE (0xFF = null)
         uint8_t received_cargo_type[3]; // 0xE0 (0xFF = null)
```

Comment:
This isn't your mistake it was made ages ago but there is a padding missing after this field. Please add `uint8_t pad_E3`. That will fix your issues as well.

Reply:
Haha, this actually fixes #412.